.github/workflows/run-eval-pf-pipeline.yml

```
@@ -3,6 +3,11 @@ name: Test Prompts with Promptflow
 
 on:
   workflow_dispatch:
+    inputs:
+      search_index:
+        description: 'Search index to be used' # description of the input
+        required: true
+        default: 'patents' # default value if not specified
```

```suggestion
        default: 'patents' # default value
```
this is the value that appears in the UI